Jenkinsfile

```diff
@@ -0,0 +1 @@
+buildPlugin()
```

Comment:
You can also set up testing against the 1.121.x branch so that we discover compatibility issues

```
buildPlugin(jenkinsVersions: [null, '2.121.1'])
```

Reply:
I did so.